src/lib/operators/import_binary.cpp

```diff
@@ -189,8 +189,8 @@ std::shared_ptr<DictionaryColumn<T>> ImportBinary::_import_dictionary_column(std
                                                                              ChunkOffset row_count) {
   const auto attribute_vector_width = _read_value<AttributeVectorWidth>(file);
   const auto dictionary_size = _read_value<ValueID>(file);
-  const auto dictionary = _read_values<T>(file, dictionary_size);
-  const auto attribute_vector = _import_attribute_vector(file, row_count, attribute_vector_width);
+  auto dictionary = _read_values<T>(file, dictionary_size);
+  auto attribute_vector = _import_attribute_vector(file, row_count, attribute_vector_width);
```

Comment:
Wait, why did these have to be changed?

Reply:
Didn't have anything to do with this PR, but I didn't want to open another two-line PR. You cannot move `const` things.